src/test/java/uk/gov/hmcts/reform/pip/data/management/service/filegeneration/nonstrategic/RptEasternWeeklyHearingListFileConverterTest.java

```diff
@@ -0,0 +1,214 @@
+package uk.gov.hmcts.reform.pip.data.management.service.filegeneration.nonstrategic;
```

Comment:
This 5 RPT tests are using the same test files. The only thing different is the list type. I think it might be better to have a single test class and use ParameterizedTest with EnumSource with loop through each list type in the test

Reply:
done.